test/dotnet-uninstall.Tests/Windows/RegistryManagerTests.cs

```diff
@@ -0,0 +1,252 @@
+﻿using FluentAssertions;
+using Microsoft.DotNet.Tools.Uninstall.Windows;
+using Xunit;
+
+namespace Microsoft.DotNet.Tools.Uninstall.Tests.Windows
+{
+    public class RegistryManagerTests
+    {
+        private static readonly string MicrosoftCorporationPublisher = "Microsoft Corporation";
+        private static readonly string X64DisplayNameFormat = "Microsoft .NET Core SDK {0}.{1}.{2} (x64)";
+        private static readonly string X86DisplayNameFormat = "Microsoft .NET Core SDK {0}.{1}.{2} (x86)";
+
+        [Fact]
+        public void TestIsDotNetCoreSdkDisplayName()
+        {
+            RegistryManager.IsDotNetCoreSdkDisplayName(string.Format(X64DisplayNameFormat, 2, 2, 300))
+                .Should().BeTrue();
+        }
+
+        [Fact]
+        public void TestIsDotNetCoreSdkPublisher()
+        {
+            RegistryManager.IsDotNetCoreSdkPublisher(MicrosoftCorporationPublisher)
+                .Should().BeTrue();
+        }
+
+        [Fact]
+        public void TestIsDotNetCoreSdkDisplayNameNull()
+        {
+            RegistryManager.IsDotNetCoreSdkDisplayName(null)
+                .Should().BeFalse();
```

Comment:
you can use `[Theory]` to condense these like https://github.com/dotnet/sdk/blob/f0a1246be1ebe11a88097ac3dd5736f0ad15b855/src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantBuildsToBeIncremental.cs#L23

Reply:
done